packages/upward-js/lib/resolvers/ServiceResolver.js

```diff
@@ -101,9 +101,12 @@ class ServiceResolver extends AbstractResolver {
                     return { data };
                 }
             })
-            .catch(e => {
+            .catch(async e => {
```

Comment:
Worth noting the change here: because we're now doing more than just a `new Error` (actually doing async work) there is stuff here that could throw with a non-descriptive error.

Example: Imagine that `await parsedQuery.render()` or `require('graphql/language/printer')` throw. The error message thrown will no longer have `ServiceResolver: Request to ${endpoint.toString()} failed`. You might want to add either another `catch` or a `try/catch`

Reply:
@zetlen another for you 😉